### PR TITLE
Feature/cohere LLM

### PR DESCRIPTION
**Pull request by dSupertramp**
Adding support for Cohere LLM

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -13,6 +13,7 @@ python-dotenv = "^1.0.0"
 pandas = "^2.0.1"
 astor = "^0.8.1"
 openai = "^0.27.5"
+cohere = "*"
```
Comment (avelino) [inline pyproject.toml:16]: better pin the version

File: pandasai/llm/cohere.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    presence_penalty: float = 0.6
+    stop: str = None
+
+
+    def __init__(
+        self,
+        api_token: str = None,
+        **kwargs,
+    ):
+        self.api_token = api_token or os.getenv("COHERE_API_KEY")
```
Comment (avelino) [inline pandasai/llm/cohere.py:40]: put the name of the environment variable in the [example file](https://github.com/gventuri/pandas-ai/blob/main/.env.example)

File: pyproject.toml (hunk 1 above)
Comment (dSupertramp) [reply pyproject.toml:16]: Still not working

**Comment by gventuri** [comment]
Hey @dSupertramp, did you have a chance to look at the failing tests?

**Comment by gventuri** [comment]
Closing, seems a duplicate of #88

**Pull request closed without merging**